src/mocks/js_dependencies.cljs

```diff
@@ -154,10 +154,7 @@ globalThis.__STATUS_MOBILE_JS_IDENTITY_PROXY__ = new Proxy({}, {get() { return (
 (def net-info #js {})
 (def touchid #js {})
 (def react-native-image-viewing #js {:default {}})
-(def safe-area-context
-  (clj->js {:SafeAreaProvider      {:_reactNativeIphoneXHelper {:getStatusBarHeight (fn [])}}
-            :SafeAreaInsetsContext {:Consumer (fn [])}
-            :SafeAreaView          {}}))
+(def safe-area-context #js {:default {}})
```

Comment:
hi @flexsurfer, thank you for the great work.
Should we rename this to `static-safe-area-insets`

Reply:
it doesn't matter, but only just to have less question to it, i'll rename it :)